longclaw/checkout/utils.py

```diff
@@ -23,6 +25,17 @@ def create_order(email,
     Create an order from a basket and customer infomation
     """
     basket_items, current_basket_id = get_basket_items(request)
+
+    if not len(basket_items) > 0:
```

Comment:
This looks like you're trying to solve a different problem. If this is the case, open an issue on the upstream and create a fork to solve it. Let's not mesh the issues.